README.md

```diff
@@ -29,228 +30,159 @@ $> yo easy-ui5 project
  ´   `  |° ´ Y `
 ```
 
-![generation flow](./docs/embeddedUsage.gif)
+## Subgenerators
 
-Run you can use `npm start` (or `yarn start`) to start the local server for development.
+This generator is split up into multiple subgenerators to help with recurring tasks even after the initial project generation. See the following list for all available subgenerators:
```

Comment:
I'm missing the new `qunit` and `opa5` subgenerators in the list 👀

Reply:
Well spotted. I added it.